package sxg;
message MessageInfoBase{
    optional int32 ver=1;
    optional string publisher=2;
    optional int32 sublisher=5;
    optional int32 message_id=3;
    optional int32 publish_time=4;
    optional string content=6;
}

